exercises/toRoman/toRoman.js

```diff
@@ -15,14 +15,88 @@
  * @returns {string} The input integer's Roman numeral representation
  */
 function toRoman(num) {
-  // This is your job. :)
+  let ones = [0,'I','II','III','IV', 'V','VI','VII','IIX','IX']
+  let ten = 'X';
+  let fifty = 'L';
+  let hundreds = 'C';
+  let fiveHundred = 'D';
+  let thousands = 'M';
+  
+  let finalString = '';
+
+  if (num === 0) {
+    return ones[0];
+  }
+
+//function toRomanThousands(num)
+  if (num >= 1000) {
+    let amountOfThousands = num / 1000
+    amountOfThousands = Math.floor(amountOfThousands)
+    finalString = finalString + thousands.repeat(amountOfThousands)
```

Comment:
You can simplify by doing `finalString += thousands.repeat(amountOfThousands)` which is the same thing, and it means increase `finalString` by `thousands.repeat(amountOfThousands)`. There's more instances of this below this line.